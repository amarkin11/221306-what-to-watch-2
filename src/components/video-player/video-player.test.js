import React from "react";
import renderer from "react-test-renderer";
import videoPlayer from "./video-player";

it(`video player correctly renders after relaunch`, () => {
  const filmsListMock = [
    {
      name: `Macbeth`,
      src: `img/macbeth.jpg`,
      preview: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
    }
  ];

  const tree = renderer
    .create(<videoPlayer
      data={filmsListMock}
      isPlaying={false}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
