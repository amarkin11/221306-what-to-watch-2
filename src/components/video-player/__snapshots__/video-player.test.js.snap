// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`video player correctly renders after relaunch 1`] = `
<videoPlayer
  data={
    Array [
      Object {
        "name": "Macbeth",
        "preview": "https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm",
        "src": "img/macbeth.jpg",
      },
    ]
  }
  isPlaying={false}
/>
`;
